CacheMultimap.adoc

```diff
@@ -0,0 +1,79 @@
+= Native cache Mutimap
+
+The goal is to provide Infinispan Native CacheMultimap support. The proposal here is to start with the smallest interface implementation, in order to provide a proper API to the first multimap client, vert-x project.
+However, this API will evolve, and this first implementation must work on Embedded and Server mode.
+This first Multimap won't support duplicate values on the same key. 
+
+[source, java]
+.CacheMultimap.java
+----
+public interface CacheMultimap<K, V> {
+   boolean put(K key, V value); // <1> <2>
+
+   Collection<V> get(K key); // <3>
```

Comment:
vert-x needs a MultiMap that won't support duplicates on a key, but other existing multimap implementations allow duplicates. The question is : do we want and need duplicates ? As far as I remember @wburns considers we should be able to support duplicates. Honestly, I don't know if we should or shouldn't, but I guess we could handle both implementations in the future, so Collection fits better on the API.